app/test_main.py

```diff
@@ -1,3 +1,61 @@
+import pytest
+
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+        (-10, -100, [0, 0])
```

Comment:
The checklist also states: 'if function receives data out of normal range, such as negative numbers, zero, or really large numbers'. It's great that you have a test case with negative numbers. However, you don't have a test case with really large numbers. Please, add it.